sklearn/feature_selection/_sequential.py

```diff
@@ -0,0 +1,218 @@
+        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
+        for more details.
+
+    pre_dispatch : int, or str, default=None
+        Controls the number of jobs that get dispatched during parallel
+        execution. Reducing this number can be useful to avoid an
+        explosion of memory consumption when more jobs get dispatched
+        than CPUs can process. This parameter can be:
+
+            - None, in which case all the jobs are immediately
+              created and spawned. Use this for lightweight and
+              fast-running jobs, to avoid delays due to on-demand
+              spawning of the jobs
+
+            - An int, giving the exact number of total jobs that are
+              spawned
+
+            - A str, giving an expression as a function of n_jobs,
+              as in '2*n_jobs'
+
+    Attributes
+    ----------
+    n_features_to_select_ : int
+        The number of features that were selected. It corresponds to
+        `n_features_to_select` unless the parameter was None.
+
+    support_ : ndarray of bool of shape (n_features,)
+        The mask of selected features.
+
+    Examples
+    --------
+    >>> from sklearn.feature_selection import SequentialFeatureSelector
+    >>> from sklearn.neighbors import KNeighborsClassifier
+    >>> from sklearn.datasets import load_iris
+    >>> X, y = load_iris(return_X_y=True)
+    >>> knn = KNeighborsClassifier(n_neighbors=3)
+    >>> sfs = SequentialFeatureSelector(knn, n_features_to_select=3)
+    >>> sfs.fit(X, y)
+    SequentialFeatureSelector(estimator=KNeighborsClassifier(n_neighbors=3),
+                              n_features_to_select=3)
+    >>> sfs.get_support()
+    array([ True, False,  True,  True])
+    >>> sfs.transform(X).shape
+    (150, 3)
+
+    See also
+    --------
+    RFE : Recursive feature elimination selection of the best number of
+          features
+    SelectFromModel : Feature selection based on importance weights
+    """
+    def __init__(self, estimator, n_features_to_select=None, forward=True,
+                 scoring=None, cv=5, n_jobs=None, pre_dispatch='2*n_jobs'):
+
+        self.estimator = estimator
+        self.n_features_to_select = n_features_to_select
+        self.forward = forward
+        self.pre_dispatch = pre_dispatch
+        self.scoring = scoring
+        self.cv = cv
+        self.n_jobs = n_jobs
+
+    def fit(self, X, y):
+        """Learn the features to select.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            Training vectors.
+        y : array-like of shape (n_samples,)
+            Target values.
+
+        Returns
+        -------
+        self : object
+        """
+
+        tags = self._get_tags()
+        X, y = self._validate_data(
+            X, y, accept_sparse="csc",
+            ensure_min_features=2,
+            force_all_finite=not tags.get('allow_nan', True),
+            multi_output=True
```

Comment:
Should this be based on the tag of the underlying estimator as well?

Reply:
I guess it should... for now we can keep it consistent with RFE?

Reply:
Okay